product_pricelist_simulation/tests/test_product_pricelist_simulation.py

```diff
@@ -86,8 +86,8 @@ def test_pricelist_simulation_product_template(self):
                 active_model="product.template", active_id=self.template_1.id
             )
         )
-        self.assertAlmostEqual(wizard_form.line_ids._records[0]["price"], 50.0)
-        self.assertAlmostEqual(wizard_form.line_ids._records[1]["price"], 80.0)
+        self.assertAlmostEqual(wizard_form.line_ids._records[0]["price"], 80.0)
+        self.assertAlmostEqual(wizard_form.line_ids._records[1]["price"], 50.0)
         # # Template 2
```

Comment:
Why this changes ?

Reply:
The sorting order for the pricelist has been changed in Odoo
v16: https://github.com/odoo/odoo/blob/16.0/addons/product/models/product_pricelist.py#L12
v17: https://github.com/odoo/odoo/blob/17.0/addons/product/models/product_pricelist.py#L13
which causes the order of wizard lines to be sorted.